.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+# Bank Account
+Congratulations! You're submitting your assignment.
+
+## Comprehension Questions
+
+|  Question 	|   Answer	|
+|---	|---	|
+|   Why is it useful to put classes inside modules?	|   	|
```

Comment:
This may seem obvious to the students, i'm not sure.... but I wonder if we should add "Why _do you think_".... to these questions to clarify that folks don't need to have all of the answers.